ZLImageEditor.xcodeproj/project.pbxproj

```diff
@@ -289,7 +289,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
Why upgrade the minimum supported version from 9.0 to 12.0?

Reply:
That was Xcode 13.3.1 and it's recommended settings. I can revert it if you still want to support iOS 9

Reply:
Well, please revert the change of this line.

Reply:
Done